source/documentation/information/mojgithubenteprise.html.md.erb

```diff
@@ -0,0 +1,8 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: MoJ GitHub Enteprise Management
+last_reviewed_on: 2023-08-02
+review_in: 6 months
+---
+
+# MoJ GitHub Enteprise Management
```

Comment:
typo: enterprise and in the filename as well

Reply:
fixed